module-5.md

```diff
@@ -12,36 +12,102 @@ Planned:      1.25 hour
 
 ## 1: Topic introduction
 
-
+* Take up homework 
+* Discuss the need for keeping a pulse of the network
+  * What attributes are important to monitor
+  * How are these attributes measured?
+  * Identify units and what they mean
+  * Where have we seen it in our daily lives?
 
 ## 2: Hands-on and material exploration
 
-Monitor flow of network traffic using different tools as we do everyday digital things.
+Monitor flow of network traffic using different tools as we do every day digital things.
 
 ### Objectives
 
+Learn how these metrics affect us in our day to day lives, and in contrast how the same metrics affect others.
 
 ### Materials
 
+* Your cellphone
+* Some sort of public WiFi or some sort of internet connection
+* A Raspberry Pi 3 for each member labeled with the SSID for the node
+* SD cards with pre-flashed images with all required software
```

Comment:
Be more specific with `pre-flashed images with all required software`

Reply:
Agreaed: Waiting for dust to settle on  the software part. I'm assuming were still using mesh orange.

Reply:
It is an option, agree on waiting for dust to settle, but also okay with using Raspbian based image if flashing is easy.

Reply:
I just don't want to be flashing images throughout the workshop. I'd rather your one stock image whatever that image is.